handlers.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"encoding/json"
+	"github.com/gorilla/mux"
+	"gopkg.in/mgo.v2/bson"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"time"
+)
+
+func Auth(fn http.HandlerFunc) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		// TODO Handle auth.
+		// Only continue if user is authorized
+		fn(w, r)
+	}
+}
```

Comment:
No it's not. I put that there for starting the auth check. Is there a way to remove that part from git but leave it locally? Besides just copying and pasting